test/bls_test.js

```diff
@@ -1,146 +1,110 @@
-const bls = require("../")
-const mcl = require("mcl-wasm")
 const assert = require("chai").assert
+const keccak256 = require("keccak256")
+const mcl = require("mcl-wasm")
+
+const bls = require("../")
 
 describe("bls", () => {
 	before(async() => {
-		await mcl.init(mcl.BLS12_381)
+		await bls.init()
 	})
 
-	it("should create base point for G1", () => {
-        let q = bls.Q()
-        assert(!q.isZero(), "did not create G1 base point")
+	it("should create generator for G1", () => {
+    const g = bls.g1()
+    assert(!g.isZero(), "did not create G1 generator")
 	})
 
 	it("should hash message to G2", () => {
-		let domain = 0
-		let message = '0x6d657373616765'
-		console.log(bls.hash_to_G2(message, domain))
+		const domain = 0
+		const message = keccak256(Buffer.from('6d657373616765', 'hex'))
+		assert(bls.hashToG2(message, domain))
 	})
```

Comment:
this is fine for now but we eventually need to make sure it conforms to the spec test vectors

Reply:
Yeah, I was thinking that the code is still in a proto-working stage, this PR is mainly to support the input/output types that lodestar is expecting, so we can start using this library (even if the logic is not quite solid yet).

Reply:
yep, that is reasonable.